docs/manage-messaging-metadata-using-gravitino.md

```diff
@@ -0,0 +1,307 @@
+    "This is a Kafka catalog",
+    properties);
+// ...
+```
+
+</TabItem>
+</Tabs>
+
+Currently, Gravitino supports the following catalog providers:
+
+| Catalog provider | Catalog property                                                |
+|------------------|-----------------------------------------------------------------|
+| `kafka`          | [Kafka catalog property](./kafka-catalog.md#catalog-properties) |
+
+### Load a catalog
+
+Refer to [Load a catalog](./manage-relational-metadata-using-gravitino.md#load-a-catalog)
+in relational catalog for more details. For a messaging catalog, the load operation is the same.
+
+### Alter a catalog
+
+Refer to [Alter a catalog](./manage-relational-metadata-using-gravitino.md#alter-a-catalog)
+in relational catalog for more details. For a messaging catalog, the alter operation is the same.
+
+### Drop a catalog
+
+Refer to [Drop a catalog](./manage-relational-metadata-using-gravitino.md#drop-a-catalog)
+in relational catalog for more details. For a messaging catalog, the drop operation is the same.
+
+### List all catalogs in a metalake
+
+Please refer to [List all catalogs in a metalake](./manage-relational-metadata-using-gravitino.md#list-all-catalogs-in-a-metalake)
+in relational catalog for more details. For a messaging catalog, the list operation is the same.
+
+### List all catalogs' information in a metalake
+
+Please refer to [List all catalogs' information in a metalake](./manage-relational-metadata-using-gravitino.md#list-all-catalogs-information-in-a-metalake)
+in relational catalog for more details. For a messaging catalog, the list operation is the same.
+
+## Schema operations
+
+`Schema` is a logical grouping of topics in a messaging catalog, if the messaging system does not support topics grouping, 
+schema operations are not supported but a "default" schema will be automatically created to include all topics
+
+:::caution note
+Gravitino currently only supports the Kafka catalog. Since Kafka does not support topic grouping, only list and load operations are supported for schema.
+:::
+
+### Create a schema
+
+Please refer to [Create a schema](./manage-relational-metadata-using-gravitino.md#create-a-schema)
+in relational catalog for more details. For a messaging catalog, the create operation is the same.
+
+### Load a schema
+
+Please refer to [Load a schema](./manage-relational-metadata-using-gravitino.md#load-a-schema)
+in relational catalog for more details. For a messaging catalog, the load operation is the same.
+
+### Alter a schema
+
+Please refer to [Alter a schema](./manage-relational-metadata-using-gravitino.md#alter-a-schema)
+in relational catalog for more details. For a messaging catalog, the alter operation is the same.
+
+### Drop a schema
+
+Please refer to [Drop a schema](./manage-relational-metadata-using-gravitino.md#drop-a-schema)
+in relational catalog for more details. For a messaging catalog, the drop operation is the same.
```

Comment:
Is the scheme in the Kafka catalog read-only?

Reply:
yes. see https://github.com/datastrato/gravitino/pull/3106/files#diff-656563ad28ab24f645bb63b4974d71f2d7c115a3547aacb44ea679fa75758ec3R34